doc/manual/manual.tex

```diff
@@ -2886,9 +2896,21 @@ \subsection{Overview}
 ...
 \end{lstlisting}
 
-This output was produced by a parameter file that, among other settings,
-contained the following values (we will discuss many such input files in
-Section~\ref{sec:cookbooks}:
+The output starts with a header that lists the used \aspect{}, \dealii{}, and
+\pfrst{} versions as well as compiler settings and the number of parallel
```

Comment:
With compiler settings you mean whether it is debug or optimized? If it is just that, then you might just want to say that.